acvm/src/lib.rs

```diff
@@ -216,15 +216,15 @@ pub enum Language {
     PLONKCSat { width: usize },
 }
 
-pub fn hash_constraint_system(cs: &Circuit) -> [u8; 32] {
+pub fn hash_constraint_system(cs: &Circuit) -> u32 {
```

Comment:
This is no longer a hash, could you create a new function called `checksum_constraint_system` instead of modifying this one?

(A checksum and a hash serve two different purposes)

Reply:
makes sense 😄